service/league_service/typedefs.py

```diff
@@ -22,6 +22,7 @@ class InvalidScoreError(LeagueServiceError):
     pass
 
 
+GameID = int
 PlayerID = int
```

Comment:
You could use a `NewType` if you wanted the type checker to be able to actually distinguish between the two types and give you a warning if you try to use a variable with type `GameID` somewhere that a `PlayerID` is expected and vice versa.

https://docs.python.org/3/library/typing.html#newtype